src/model/utilities/Player.test.js

```diff
@@ -0,0 +1,27 @@
+import Player from './Player';
+
+describe('Player', () => {
+  const player = Player();
+  const mockCallback = jest.fn();
+  const testQuestion = {
+    answers: ["Declan's men and his cook.", 'Los Pollos driver', 'No-Doze', 'Max Arciniega'],
+    correctAnswer: 'NoDoze',
+    questionObject: 'Beaten to death.',
+  };
+
+  it('callback powinien zostac wywołany raz jesli zadamy graczowi pytanie', () => {
+    player.askQuestion(mockCallback, testQuestion);
+
+    expect(mockCallback.mock.calls.length).toBe(1);
+  });
+
+  it('callback powinien zostac wywolany jesli gracz odpowie', () => {
+    player.answer(mockCallback, testQuestion);
+
+    expect(mockCallback.mock.calls.length).toBe(2);
```

Comment:
Czy ten test  na pewno sprawdza nam poprqawnie? Coś trzeba tu naprawić. Przecież odpowiedziałeś tylko raz.
To dlaczego wywołany jest 2 razy?

Wrzuć mocka do bloku z testem ;)

Reply:
Jeżeli wyciągne mockCallback poza funkcję najpierw wykona się w pierwszym it, a następnie w drugim it, dlatego jest 2.
Jeżeli wsadzę const mockCallback = jest.fn(); do każdego it, to faktycznie toBe będzie równe 1 w każdym it